go.mod

```diff
@@ -65,7 +62,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.6.1
 	github.com/vishvananda/netlink v1.1.1-0.20200210222539-bfba8e4149db
-	go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20190911215424-9ed5f76dc03b
```

Comment:
This change is reverting PR https://github.com/cilium/cilium/pull/10381, is this intended?

After pulling this PR, I can't re-add this dependency back to `go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738`.

Reply:
no it's not intentional, i just ran `go mod vendor`. i'll look into it.

Reply:
i think i know what's going on. fix coming soon